src/test/java/seedu/notus/util/parser/ParserManagerTest.java

```diff
@@ -61,6 +66,163 @@ void parseCommand_invalidAddNoteString_returnIncorrectCommand() {
         assertTrue(command instanceof IncorrectCommand);
     }
 
+    //@@author R-Ramana
+    @Test
+    void parseCommand_validHelpString_returnHelpCommand() {
+        inputString = "help";
+        command = parserManager.parseCommand(inputString);
+        assertTrue(command instanceof HelpCommand);
+
+        inputString = "help me please";
+        command = parserManager.parseCommand(inputString);
+        assertTrue(command instanceof HelpCommand);
+    }
+
+    @Test
+    void parseCommand_validFindNote_returnFindCommand() {
+        inputString = "find-n note";
+        command = parserManager.parseCommand(inputString);
+        assertTrue(command instanceof FindCommand);
+
+        inputString = "find-n hola my name is John";
+        command = parserManager.parseCommand(inputString);
+        assertTrue(command instanceof FindCommand);
+
+        inputString = "find-n /i 1 /t hey";
+        command = parserManager.parseCommand(inputString);
+        assertTrue(command instanceof FindCommand);
+    }
+
+    @Test
+    void parseCommand_invalidArchiveNote_returnIncorrectCommand() {
+        inputString = "archive-n /i -20";
+        command = parserManager.parseCommand(inputString);
+        assertTrue(command instanceof IncorrectCommand);
+
+        inputString = "archive /i 2";
+        command = parserManager.parseCommand(inputString);
+        assertTrue(command instanceof IncorrectCommand);
+
+        inputString = "archive /t hola";
+        command = parserManager.parseCommand(inputString);
+        assertTrue(command instanceof IncorrectCommand);
```

Comment:
This 2 should not be tested here as there is a function that checks for invalid string. You can move these 2 there. For the test here all command word, "archive-n", should be correct. It should be the parameter that varies which will result in the IncorrectCommand being return. Cases to consider:

archive-n /i 0
archive-n (no parameter)